templates/_components/_pageBuilder.twig

```diff
@@ -0,0 +1,29 @@
+{# _components/_pageBuilder #}
+
+
+
+{# Params #}
+{# @var builderComponents \craft\elements\MatrixBlock #}
+{% set builderClass = params.class ?? '' %}
+{% set builderComponents = params.components.all() ?? [] %}
+
+
+
+{# Render #}
+{% for component in builderComponents %}
+
+  {% if loop.first and builderClass %}
+    <div class="{{ builderClass }}">
+  {% endif %}
+
+  {% include '_routers/component.twig' with {
+    entry: entry,
+    params: component|merge({
+      type: component.type()
```

Comment:
This entire `merge` appears redundant.

Reply:
Well, if you look at it out of context that is the consequence. As I remember, without the addition of that merge, the component router doesn't function correctly. Whether the 'fix' should be moved from the page builder to the component router, I don't know.

Reply:
We can remove it now and wait for the error to crop up again in the future.

Reply:
What was the error? Can you try reproducing it on a live project?

Reply:
IIRC the error was that the component router checks for `type.handle` and throws an error because `type` is lazy-loaded.

Reply:
I think this was used on thirdeye, so I'll spin that up locally and see if I can reproduce it.